tests/test_serialization.py

```diff
@@ -317,6 +277,21 @@ def test_experiment_serialization():
 
     e.add(0, Detector.ppnr(24, 4))
     e.add(1, Detector.threshold())
+
+    mzi = catalog['mzi phase last'].build_circuit()
+    ffc = FFConfigurator(2, 0, mzi, {"phi_a": 0, "phi_b": 0}, "ffc name")
+    ffc.add_configuration((0, 1), {"phi_a": 3, "phi_b": 0})
+    ffc.add_configuration((1, 0), {"phi_a": 0, "phi_b": 3})
+    e.add(0, ffc)
+
+    ffcp = FFCircuitProvider(1, 0, Circuit(2) // BS.H(), "ffcp name")
+    ffcp.add_configuration((0,), Circuit(2) // BS.Rx())
+    subexp = Experiment(2)
```

Comment:
Does it work if we try to add a `Processor` here as it is the case in pereval 0.12 ?
My take is that the `Configurator` should extract the experiment from the `Processor`